### ISSUE TITLE
How to fix data_range in ssim ?

### ISSUE THREAD

**Issue by ChafikBak** [VainF/pytorch-msssim#14]
Hi @VainF 

I am trying to train my model with ssim loss.

Minimal and maximal values of my data are:
min_val=-0.2511819651191651
max_val=336.32888595625667

I used 2 methods for training:

**Method 1:**
```
output_normalized = (output-min_val)/(max_val-min_val)
target_normalized = (target-min_val)/(max_val-min_val)
loss = 100*(1 - ssim(output_normalized, target_normalized,  data_range=1, size_average=True))
```
**Method 2:**
`loss = 100*(1 - ssim(output-min_val, target-min_val,  data_range=max_val-min_val, size_average=True))`

The first method gives very noisy results. Do you have some explanation please?
